##信息传输完成的标志位
std_msgs/String collection_flag  
uint8 point_n
##目标点到达的标志位
bool arrive_flag